src/fides/api/api/v1/endpoints/privacy_experience_endpoints.py

```diff
@@ -152,8 +183,15 @@ def privacy_experience_list(
         )
 
     if component is not None:
+        # If searching for "overlay" - return both overlay types: the regular overlay and the TCF Overlay,
```

Comment:
But we can return both here, that was why I added the code comment!  The frontend requested that the backend be a little more generous about what it meant when we filtered on "component=overlay".  Because the frontend doesn't know whether to filter on `overlay` or `tcf_overlay`, it just knows where it needs some sort of overlay.    

So here, I am basically excluding the privacy center experience from being returned.  And then later, it is the presence of the `region` filter that further restricts whether a TCF or regular overlay is returned.